src/Codeception/Module/Symfony.php

```diff
@@ -284,10 +294,12 @@ protected function getKernelClass()
 
         require_once $file;
 
-        $possibleKernelClasses = [
-            'AppKernel', // Symfony Standard
-            'App\Kernel', // Symfony Flex
-        ];
+        if (empty($this->config['kernel_class'])) {
+            $possibleKernelClasses = static::POSSILE_KERNEL_CLASSES;
+        } else {
+            $possibleKernelClasses = $this->config['kernel_class'];
```

Comment:
`$possibleKernelClasses` is an array, `kernel_class is a string` and it would look odd as an array in the config, so I think that this line would be better writter as 
```php
$possibleKernelClasses = [ $this->config['kernel_class'] ];
```

Also it would be good to do some typechecking and throw ModuleException if kernel class is not a string. (unless you want to accept arrays, then typechech that too).

Reply:
Done